.goreleaser.yml

```diff
@@ -68,19 +93,43 @@ nfpms:
       - apk
       - deb
       - rpm
-brews:
-  - name: descope
-    url_template: "https://github.com/descope/descopecli/releases/download/{{ .Tag }}/{{ .ArtifactName }}"
-    directory: Formula/d/
-    homepage: https://www.descope.com/
-    description: A command line utility for working with the Descope management APIs
-    license: "MIT"
-    skip_upload: true
-    repository:
-      owner: descope
-      name: homebrew
+
 release:
   draft: true
   replace_existing_draft: true
   replace_existing_artifacts: true
-report_sizes: true
+
+changelog:
+  sort: asc
+  use: github
+  filters:
+    exclude:
+      - "^test:"
```

Comment:
not very familiar with goreleaser - but do we want to force semantic commits like we do in descope-js for this purpose?

Reply:
Doesn't force
Just organizes changelog if they're there
Otherwise they'll go at the bottom

The one big good thing is the filter out of chores which will tone down renovate noise from changelog